src/app/faqs/faq-list/faq-list.component.html

```diff
@@ -0,0 +1,154 @@
+    <button
+      class="mb-2"
+      mat-button
+      color="warn"
+      (click)="onClickCategory(null)"
+      *ngIf="faqCategories.length - getFilteredCategories().length"
+    >
+      {{ faqCategories.length - getFilteredCategories().length }} Category
+      hidden
+    </button>
+
+    <!-- Categories Accordian -->
+    <mat-accordion class="w-100">
+      <mat-expansion-panel
+        *ngFor="let faqCategory of getFilteredCategories(); let i = index"
+        (opened)="onOpenFaqCategory(faqCategory)"
+        [expanded]="
+          faqCategory.id === filterFaqCategory?.id ||
+          faqCategory.id === selectedFaqCategory?.id
+        "
+      >
+        <mat-expansion-panel-header>
+          <mat-panel-title>
+            {{ faqCategory.title }}
+          </mat-panel-title>
+          <mat-panel-description class="w-100">
+            {{ faqCategory.description }}
+          </mat-panel-description>
+        </mat-expansion-panel-header>
+
+        <!-- FAQ Accordian -->
+        <div class="mt-2">
+          <mat-accordion>
+            <mat-expansion-panel
+              *ngFor="let faq of findAllFaqsByFaqCategory(faqCategory)"
+              [expanded]="selectedFaq?.id === faq.id"
+              (opened)="onOpenFaq(faq)"
+            >
+              <mat-expansion-panel-header>
+                <mat-panel-title>
+                  {{ faq.title }}
+                </mat-panel-title>
+              </mat-expansion-panel-header>
+              <p class="ml-4">{{ faq.description }}</p>
+
+              <!-- FAQ Admin Toolbar -->
+              <mat-divider *ngIf="userIsAdmin" class="mb-1"></mat-divider>
+              <div
+                class="d-flex flex-row justify-content-end"
+                *ngIf="userIsAdmin"
+              >
+                <button
+                  mat-icon-button
+                  color="primary"
+                  (click)="onClickStarFaq(faq)"
+                >
+                  <mat-icon>
+                    {{ faq.starred ? "star" : "star_border" }}
+                  </mat-icon>
+                </button>
+                <button mat-icon-button (click)="onClickEditFaq(faq)">
+                  <mat-icon>edit</mat-icon>
+                </button>
+                <button
+                  mat-icon-button
+                  color="warn"
+                  (click)="onClickDeleteFaq(faq)"
+                >
+                  <mat-icon>delete</mat-icon>
+                </button>
+              </div>
+            </mat-expansion-panel>
+          </mat-accordion>
+        </div>
+
+        <!-- Category Admin Toolbar -->
+        <div
+          class="d-flex flex-row justify-content-end mt-3"
+          *ngIf="userIsAdmin"
+        >
+          <button
+            *ngIf="userIsAdmin"
+            mat-raised-button
+            color="primary"
+            class="mr-2 w-100"
+            (click)="onClickAddFaq(faqCategory)"
+          >
+            Add FAQ
```

Comment:
have this be a FAB instead - somewhere at the top. if its at the bottom and theres a ton of faqs i think it would be hard for folks to find